inferno-core/inferno-core.cabal

```diff
@@ -1,107 +1,114 @@
-cabal-version:       2.4
-name:                inferno-core
-version:             0.11.6.0
-synopsis:            A statically-typed functional scripting language
-description:         Parser, type inference, and interpreter for a statically-typed functional scripting language
-category:            DSL,Scripting
-homepage:            https://github.com/plow-technologies/inferno.git#readme
-bug-reports:         https://github.com/plow-technologies/inferno.git/issues
-copyright:           Plow-Technologies LLC
-license:             MIT
-license-file:        LICENSE
-author:              Sam Balco
-maintainer:          info@plowtech.net
-build-type:          Simple
+cabal-version:      2.4
```

Comment:
I `cabal-fmt`'d the Cabal files, the inconsistency was bugging me. I can add a formatting check for this in the future re-formatting PR